Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk
```

Comment:
Also, the base `openjdk` image is *much* heavier than we want. I strongly recommend using the variant based on Alpine Linux, which is probably about a tenth the size.